### PR TITLE
docs(scss): refactor panel less dev doc styles to scss

### PR DESCRIPTION
**Pull request by BY00565233**
## Description

Refactor Panel Dev Docs from LESS to SCSS

### What are the relevant story cards/tickets? Any additional PRs or other references?

Jira: SURF-2083

## Before you request a review for this PR:

- [x] For UI changes, did you manually test in recent versions of modern browsers (Chrome, Firefox, and Safari)?
- [x] For UI changes, did you manually test in IE11 and legacy Edge?
- [ ] Did you add component tests for any new code?
- [ ] Did you run the component unit tests via `yarn test` to ensure all tests passed?
- [ ] Did you include a screenshot of the component tests?
- [ ] If you changed/added functionality, did you update the demo page and documentation?
- [ ] If needed, did you add or modify the demo test page to test the changed/added functionality?
- [x] Did you assign reviewers?
- [x] In Jira, have you linked to this PR on the ticket(s)?

### CODE REVIEW

File: docs/styles/demo/panel-docs.less [hunk 1]
```diff
@@ -1,25 +1,25 @@
-.docs-panel {
-  #demo-basic {
-    height: 15rem;
-    max-height: 20rem;
-    min-height: 10rem;
+// .docs-panel {
```
Comment (100stacks) [inline docs/styles/demo/panel-docs.less:1]: Please use `git mv`.

File: docs/styles/spec/panel-spec.less [hunk 2]
```diff
@@ -1,27 +1,27 @@
-.specs-panel {
+// .specs-panel {
```
Comment (100stacks) [inline docs/styles/spec/panel-spec.less:1]: Please use `git mv`.

File: docs/components/panel/_spec.scss [hunk 3]
```diff
@@ -0,0 +1,27 @@
+.specs-panel {
```
Comment (100stacks) [inline docs/components/panel/_spec.scss:1]: Please use `git mv`.

File: docs/components/panel/_demo.scss [hunk 4]
```diff
@@ -0,0 +1,25 @@
+.docs-panel {
```
Comment (100stacks) [inline docs/components/panel/_demo.scss:1]: Please use `git mv`.

**Review by 100stacks — CHANGES_REQUESTED** [review]

File: docs/components/_index.scss [hunk 5]
```diff
@@ -1,6 +1,7 @@
 /// load styles required to demonstrate each component
 @import "box/index";
 @import "grid/index";
+@import "panel/demo";
```
Comment (100stacks) [inline docs/components/_index.scss:4]: Please rename file reference to `index`.

File: docs/components/panel/_demo.scss (hunk 4 above)
Comment (100stacks) [inline docs/components/panel/_demo.scss:1]: Please use `git mv` and rename file to `_index.scss`.

**Review by 100stacks — CHANGES_REQUESTED** [review]

File: docs/styles/demo/panel-docs.less (hunk 1 above)
Comment (BY00565233) [reply docs/styles/demo/panel-docs.less]: Changes have been done, Please review

File: docs/styles/spec/panel-spec.less (hunk 2 above)
Comment (BY00565233) [reply docs/styles/spec/panel-spec.less]: Changes have been done, Please review

File: docs/components/panel/_spec.scss (hunk 3 above)
Comment (BY00565233) [reply docs/components/panel/_spec.scss]: Changes have been done, Please review

File: docs/components/panel/_demo.scss (hunk 4 above)
Comment (BY00565233) [reply docs/components/panel/_demo.scss]: Changes have been done, Please review

File: docs/components/_index.scss (hunk 5 above)
Comment (BY00565233) [reply docs/components/_index.scss]: Changes have been done, Please review

File: docs/components/panel/_demo.scss (hunk 4 above)
Comment (BY00565233) [reply docs/components/panel/_demo.scss]: Changes have been done, Please review

**Review by lalithkarikelli — APPROVED** [review]
DEV LGTM

File: docs/docs.less [hunk 6]
```diff
@@ -171,14 +171,5 @@ min-version {
     }
   }
 }
-
-/* ---------------------------------------- *\
- * HELPERS
-\* ---------------------------------------- */
-
-// Import from docs/styles/*
```
Comment (100stacks) [inline docs/docs.less:179]: Please revert and change to `//` comments.

**Review by 100stacks — CHANGES_REQUESTED** [review]

**Review by 100stacks — APPROVED** [review]

File: docs/docs.less [hunk 7]
```diff
@@ -171,14 +171,5 @@ min-version {
     }
```
Comment (100stacks) [inline docs/docs.less:171]: Please merge and rebase `master` to resolve file conflicts:

Screen Shot 2020-11-17 at 1 41 14 PM

File: docs/components/_index.scss [hunk 8]
```diff
@@ -3,6 +3,7 @@
 @import "grid/index";
 @import "icon/index";
 @import "navigation/index";
+@import "panel/index";
```
Comment (100stacks) [inline docs/components/_index.scss:6]: Please merge and rebase `master` to resolve file conflicts:

Screen Shot 2020-11-17 at 1 41 14 PM

**Review by 100stacks — CHANGES_REQUESTED** [review]

File: docs/docs.less (hunk 7 above)
Comment (BY00565233) [reply docs/docs.less:171]: Changes have been done. Please review

File: docs/components/_index.scss (hunk 8 above)
Comment (BY00565233) [reply docs/components/_index.scss:6]: Changes have been done. Please review

**Review by 100stacks — APPROVED** [review]
Dev LGTM

**Pull request merged**